src/files/3_AllProjectsAnalyzers.editorconfig

```diff
@@ -393,13 +393,13 @@ dotnet_diagnostic.CA1200.severity = none
 # CA1310: Specify StringComparison for correctness (ex: string.Compare)
 # CA1311: Specify a culture or use an invariant version (ex: string.ToUpper)
 dotnet_diagnostic.CA1303.severity = none
-dotnet_diagnostic.CA1304.severity = suggestion
-dotnet_diagnostic.CA1305.severity = suggestion
-dotnet_diagnostic.CA1307.severity = suggestion
+dotnet_diagnostic.CA1304.severity = silent
+dotnet_diagnostic.CA1305.severity = silent
+dotnet_diagnostic.CA1307.severity = silent
 dotnet_diagnostic.CA1308.severity = none
-dotnet_diagnostic.CA1309.severity = suggestion
-dotnet_diagnostic.CA1310.severity = suggestion
-dotnet_diagnostic.CA1311.severity = suggestion
+dotnet_diagnostic.CA1309.severity = silent
+dotnet_diagnostic.CA1310.severity = silent
```

Comment:
This one also seems to detect implicit current culture comparison: https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/CA1310

Reply:
The change was reverted back suggestion. Changing it to a warning could be considered in the future